variables.tf

```diff
@@ -65,6 +65,12 @@ variable "node_version" {
   default     = ""
 }
 
+variable "master_authorized_networks_config" {
+  description = "The desired configuration options for master authorized networks. Omit the nested cidr_blocks attribute to disallow external access (except the cluster node IPs, which GKE automatically whitelists)."
```

Comment:
@pratikmallya Could we expand this to include an example config, like here: https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/10#issuecomment-423255873